deploy/001-risk-fund-converter.ts

```diff
@@ -19,19 +24,33 @@ type Addresses = {
     riskFund: string;
     poolRegistry: string;
     corePool: string;
+    poolStableCoin: string;
+    poolDeFi: string;
+    poolGameFi: string;
+    poolTron: string;
+    USDT: string;
     vBNB: string;
     wBNB: string;
     proxyOwnerAddress: string;
   };
 };
 
+let comptrollers: string[];
+let assets: string[][];
+let values: boolean[][];
+
 const addresses: Addresses = {
   bsctestnet: {
     acm: "0x45f8a08F534f34A97187626E05d4b6648Eeaa9AA",
     oracle: "0x3cD69251D04A28d887Ac14cbe2E14c52F3D57823",
     riskFund: "0xBe4609d972FdEBAa9DC870F4A957F40C301bEb1D",
     poolRegistry: "0xC85491616Fa949E048F3aAc39fbf5b0703800667",
     corePool: "0x94d1820b2D1c7c7452A163983Dc888CEC546b77D",
+    poolStableCoin: "0x10b57706AD2345e590c2eA4DC02faef0d9f5b08B",
+    poolDeFi: "0x23a73971A6B9f6580c048B9CB188869B2A2aA2aD",
+    poolGameFi: "0x1F4f0989C51f12DAcacD4025018176711f3Bf289",
+    poolTron: "0x11537D023f489E4EF0C7157cc729C7B69CbE0c97",
```

Comment:
I miss the "Pool Liquid Staked BNB". You can check the addresses here:

* mainnet: https://docs-v4.venus.io/deployed-contracts/isolated-pools#pool-liquid-staked-bnb
* testnet: https://docs-v4.venus.io/deployed-contracts/isolated-pools#pool-liquid-staked-bnb-1

Reply:
"Pool Liquid Staked BNB" does not contain USDT market so i did not add it, should i add it?